cloudant-client/src/main/java/com/cloudant/client/internal/views/MultipleRequestBuilderImpl.java

```diff
@@ -47,6 +45,8 @@ private MultipleRequestBuilderImpl(ViewQueryParameters<K, V> params, ViewMultipl
 
     public MultipleRequestBuilder<K, V> add() {
         multiViewRequester.add(this.viewQueryParameters);
+        //one at least one request has been added, then it is buildable
```

Comment:
Grammatical mistake carried over from previous comment

Reply:
Also, are lines 41 and 42 necessary?
```
         //parameters have been set, not buildable until add() is called
         isBuildable = false;
         return this;
 ```
Since `isBuildable` is set to `false` in the ctor